packages/api-client-core/package.json

```diff
@@ -28,8 +28,9 @@
     "prerelease": "gitpkg publish"
   },
   "dependencies": {
+    "@manufac/repeater": "^4.0.6",
```

Comment:
@airhorns You have been added to this repo. I'll try to move all that code to this monorepo itself via subsequent PRs.